versions.tf

```diff
@@ -4,7 +4,12 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">= 3.39.0"
+      version = "~> 3.97.0"
```

Comment:
We shouldn't require a specific version of the Terraform provider to use this module:

```suggestion
      version = ">= 3.39.0"
```

We set a minimum required version of 3.39 because of this change listed in the Azure provider changelog:

![image](https://github.com/user-attachments/assets/d7f37be1-e571-4268-875d-bb29e3ed5352)

This required version should be bumped if this module uses features that were added in a later release.